### PR TITLE
update golangci lint

### PR DESCRIPTION
**Pull request by DarjaLeonova**

### CODE REVIEW

File: .golangci.yml [hunk 1]
```diff
@@ -5,52 +5,73 @@ run:
...
+   - funlen
+   - gci
+   - ginkgolinter
+   - goconst
+   - gocheckcompilerdirectives
+   - gochecknoglobals
+   - gochecknoinits
+   - godox
+   - goerr113
+   - gofmt
```
Comment (VladislavsPerkanuks) [inline .golangci.yml:30]: might be useful

File: .golangci.yml [hunk 2]
```diff
@@ -5,52 +5,73 @@ run:
...
+   - sqlclosecheck
+   - structcheck
+   - stylecheck
+   - tagalign
+   - tagliatelle
+   - tenv
+   - testableexamples
+   - testpackage
+   - thelper
+   - tparallel
```
Comment (VladislavsPerkanuks) [inline .golangci.yml:68]: this might be usefull too

**Review by klavinsjanis — APPROVED** [review]

File: .golangci.yml [hunk 3]
```diff
@@ -5,52 +5,73 @@ run:
...
+   - usestdlibvars
+   - varcheck
+   - varnamelen
+   - wastedassign
```
Comment (VladislavsPerkanuks) [inline .golangci.yml:72]: could be useful too

File: cmd/translate/main_grpc_integration_test.go [hunk 4]
```diff
@@ -100,16 +100,16 @@ func Test_UploadTranslationFile_gRPC(t *testing.T) {
 
 	// Requests
 
-	happyRequest := randUploadRequest(t, service.Id)
+	happyRequest := randUploadRequest(t, service.GetId())
```
Comment (VladislavsPerkanuks) [inline cmd/translate/main_grpc_integration_test.go:103]: was that some new linter issue or scope creep ? 🤔

File: pkg/convert/arb_test.go [hunk 5]
```diff
@@ -233,9 +233,7 @@ func Test_ToArb(t *testing.T) {
 			t.Parallel()
 
 			actual, err := ToArb(tt.input)
-			if !assert.NoError(t, err) {
-				return
-			}
+			require.NoError(t, err)
```
Comment (VladislavsPerkanuks) [inline pkg/convert/arb_test.go:236]: was that some new linter issue or scope creep ? 🤔

File: .golangci.yml (hunk 1 above)
Comment (janishorsts) [reply .golangci.yml:30]: @VladislavsPerkanuks, it is already covered by `gofumpt`.

File: .golangci.yml (hunk 2 above)
Comment (janishorsts) [reply .golangci.yml:68]: @VladislavsPerkanuks, @DarjaLeonova , we'll review later which linters have been disabled previously but might be useful. As a separate task.

File: cmd/translate/main_grpc_integration_test.go (hunk 4 above)
Comment (DarjaLeonova) [reply cmd/translate/main_grpc_integration_test.go:103]: It was a new linter issue (protogetter)

File: pkg/convert/arb_test.go (hunk 5 above)
Comment (janishorsts) [reply pkg/convert/arb_test.go:236]: new linter https://golangci-lint.run/usage/linters/#testifylint
